src/merge-pr.js

```diff
@@ -0,0 +1,62 @@
+
+var flagutil = require('./flagutil');
+var optimist = require('optimist');
+var shelljs = require('shelljs');
+var executil = require('./executil');
+var gitutil = require('./gitutil');
+var superspawn = require('./superspawn');
+
+module.exports = function *(argv) {
+    var opt = flagutil.registerHelpFlag(optimist);
+    opt.options('pr', {
+            desc: 'PR # that needs to be merged',
+            demand: true
+        }).options('remote', {
+           desc: 'Named remote for the github apache mirror',
+           demand: true  
+        }).options('pretend', {
+            desc: 'Don\'t actually run commands, just print what would be run.',
+            type:'boolean'
+        });
+    argv = opt
+        .usage('Merges the pull request to master\n' +
+        '\n' +
+        'Usage: $0 merge-pr --pr 111 --remote mirror')
+        .argv;
+   if (argv.h) {
+        optimist.showHelp();
+        process.exit(1);
+   }
+   yield gitutil.stashAndPop("", function*() {
+       yield executil.execOrPretend(executil.ARGS('git checkout master'), argv.pretend);
+    
+       yield executil.execOrPretend(executil.ARGS('git pull origin master'), argv.pretend);
+       
+       yield executil.execOrPretend(['git', 'fetch', '-fu', argv.remote,
+            'refs/pull/' + argv.pr + '/head:pr/' + argv.pr], argv.pretend);
+    
+       try {
+            yield executil.execHelper(executil.ARGS('git merge --ff-only pr/' + argv.pr),
```

Comment:
It might be more clear to assign `'pr/' + argv.pr` to a variable, since it's used in many places.